cmd/faucet/faucet.go

```diff
@@ -166,7 +165,7 @@ func InitFaucet(logger zerolog.Logger) (Faucet, error) {
 		log:             logger,
 		TokensAvailable: float64(cfg.DailyLimit),
 		DailySupply:     float64(cfg.DailyLimit),
-		Amount:          float64(amount),
+		Amount:          float64(cfg.Amount),
```

Comment:
unnecessary conversion

```suggestion
		Amount:          cfg.Amount,
```